rotkehlchen/assets/utils.py

```diff
@@ -215,6 +215,9 @@ def get_or_create_evm_token(
         encounter: TokenEncounterInfo | None = None,
         coingecko: str | None = None,
         cryptocompare: str | None = None,
+        fallback_decimals: int | None = None,
```

Comment:
Why do this? In our usecase it's not a fallback.

In our usecase they are all the same, no ?

If provided instead just don't query the chain at all. Why query, only to see it's missing and then use the given one? That makes no sense.